src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: bold;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.firstForm {
+  margin-top: 20px;
+  position: relative;
+}
+
+.secondForm {
+  margin: 20px 0;
+  position: relative;
+}
+
+.search {
+  font-family: inherit;
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  border: 1px solid rgba(225, 231, 237, 1);
```

Comment:
rgb
```suggestion
  border: 1px solid rgb(225, 231, 237);
```

Reply:
fix it
![image](https://user-images.githubusercontent.com/98764641/210966390-1e66c4f6-924b-4d1d-a1c3-aaa9b25a3ea3.png)